src/main/java/nl/hr/recipefinder/model/dto/UserDto.java

```diff
@@ -0,0 +1,41 @@
+package nl.hr.recipefinder.model.dto;
+
+import nl.hr.recipefinder.security.Role;
+
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+
+public class UserDto {
+  private String username;
+
+  private String password;
+
+  @Enumerated(EnumType.STRING)
+  private Role role;
+
+
+
+  public String getUsername() {
```

Comment:
Can use the lombok.Data annotation (@Data) to generate getters and setters.

Reply:
Zal het even toevoegen

Reply:
@MartinDrok kun je nog eens kijken